### ISSUE TITLE
Cannot find SVA and DVA methods.

### ISSUE THREAD

**Issue by hxdflying** [chen-si-jia/Trajectory-Long-tail-Distribution-for-MOT#1]
Thank you so much for your wonderful work. Where are the SVA and DVA methods in the code, please?

**Comment by chen-si-jia** [comment]
The SVA and DVA code will be updated soon. They are not difficult to implement. If you are in a hurry, you can implement them according to the description in the paper.

**Comment by wenkebei** [comment]
Thank you so much for your wonderful work.where are Input:id_target_all_nID_MOT17_half_train.txt mean in GS/step1,please?

**Comment by chen-si-jia** [comment]
> Thank you so much for your wonderful work.where are Input:id_target_all_nID_MOT17_half_train.txt mean in GS/step1,please?

id_target_all_nID_MOT17_half_train.txt is the id label for the training data. It can be obtained by uncommenting lines 53-54, 85-86 and 120-123 of the src/lib/trains/base_trainer.py file.

**Comment by chen-si-jia** [comment]
> Thank you so much for your wonderful work.where are Input:id_target_all_nID_MOT17_half_train.txt mean in GS/step1,please?

id_target_all_nID_MOT17_half_train.txt is the id label for the training data. It can be obtained by uncommenting lines 53-54, 85-86 and 120-123 of the src/lib/trains/base_trainer.py file.

**Comment by Lsyccnu** [comment]
> > Thank you so much for your wonderful work.where are Input:id_target_all_nID_MOT17_half_train.txt mean in GS/step1,please?
> 
> id_target_all_nID_MOT17_half_train.txt is the id label for the training data. It can be obtained by uncommenting lines 53-54, 85-86 and 120-123 of the src/lib/trains/base_trainer.py file.
您好，请问这个获取是需要怎么运行base_trainer.py 文件呢？这貌似是一个封装训练方法的类？

**Comment by wwwshengbo** [comment]
你好

> > > > Thank you so much for your wonderful work.where are Input:id_target_all_nID_MOT17_half_train.txt mean in GS/step1,please?
> > > 
> > > 
> > > id_target_all_nID_MOT17_half_train.txt is the id label for the training data. It can be obtained by uncommenting lines 53-54, 85-86 and 120-123 of the src/lib/trains/base_trainer.py file.
> > > 您好，请问这个获取是需要怎么运行base_trainer.py 文件呢？这貌似是一个封装训练方法的类？
> 
> base_trainer.py 不是主函数程序，不可以单独运行。需要运行训练代码（训练指令位于experiments文件夹下），就可以运行进去。

你好，能够将模型，使用阿里云盘分享一下链接，百度云，下载不下来？

**Comment by wwwshengbo** [comment]
> SVA and DVA methods
@chen-si-jia 你好，论文提出的方法感觉非常好，想问一下这两个方法 SVA and DVA methods，大概什么时候会上传？

**Comment by chen-si-jia** [comment]
DVA代码刚已开源。SVA代码也简单，是可以从开源的DVA，加一个卡尔曼滤波，拼凑后即可。

**Comment by chen-si-jia** [comment]
The DVA code has just opened. The SVA code is also simple, can be from the opened DVA code, adding a Kalman filter. That's it.

**Comment by wwwshengbo** [comment]
> DVA代码刚已开源。SVA代码也简单，是可以从开源的DVA，加一个卡尔曼滤波，拼凑后即可。

小白入门刚跟踪，感谢感谢非常感谢

**Comment by chen-si-jia** [comment]
> > DVA代码刚已开源。SVA代码也简单，是可以从开源的DVA，加一个卡尔曼滤波，拼凑后即可。
> 
> 小白入门刚跟踪，感谢感谢非常感谢

我们论文最重要的意义是开启一个新问题，可以尝试对我们提出的这个问题进行新的探索，祝你取得一定的成绩。

**Comment by wwwshengbo** [comment]
> > > DVA代码刚已开源。SVA代码也简单，是可以从开源的DVA，加一个卡尔曼滤波，拼凑后即可。
> > 
> > 
> > 小白入门刚跟踪，感谢感谢非常感谢
> 
> 我们论文最重要的意义是开启一个新问题，可以尝试对我们提出的这个问题进行新的探索，祝你取得一定的成绩。

确实，感觉论文角度非常好，但对于跨专业，刚入门人来说，代码可能正常运行，都是个问题
